### DISCUSSION TITLE
Open Channel - Errormessage liquidity in btc while sats shown for opening

### DISCUSSION THREAD

**Discussion by arbadacarbaYK** [Ride-The-Lightning/RTL#895] [General]
Why do Errormessages to opening a channel with too less liquidity calculate the liquidity in btc while for opening a channel you need to give in sats. Quite unnessecary confusion

**Comment by ShahanaFarooqui** [comment]
RTL only captures error messages thrown by LND/Clightning/Eclair and shows them on the UI.

**Reply by saubyk** [reply]
@Hacker123237656777334523457 
Basically, the error messages that we are displaying are returned by the underlying node implementation. You can open an issue with LND to standardize the units of the amount in the errors returned to be sats.

**Comment by arbadacarbaYK** [comment]
Wouldnt it be easier to be able to switch to btc when opening a channel to overcome this ? In the end it should be a rounded UX and its not if the units dont match imo

**Reply by saubyk** [reply]
Cannot switch units like that, it creates confusion. Sats should be the unit on Lightning. I am sure this is an oversight in the error messages and can be addressed quickly by LND. Changing application logic to match the unit in error message is not a good strategy.

**Comment by arbadacarbaYK** [comment]
on top it would be nice to see the max-amount left over to open a channel instead of trial and error - in the same unit..
